server/Controllers/PointsController.cs

```diff
@@ -60,56 +110,45 @@ public IActionResult Post(Guid id, [FromBody]Point point)
                 return new BadRequestObjectResult(ex.ToString());
             }
             points[id] = point;
```

Comment:
Please either write warning to console/logs or disallow to overwrite same point GUID